pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -1112,6 +1112,14 @@ func (c *Cluster) saveMonConfig() error {
 		return errors.Wrap(err, "failed to update csi cluster config")
 	}
 
+	// Skip the new CSI-operator creation when holder pod is enabled until multus support is added in the CSI operator
```

Comment:
comment needs updating? seems like this only needs to comment on the creation of the ceph connection.

Reply:
removed